src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -86,6 +90,17 @@ class MarathonModule(conf: MarathonConf, http: HttpConf)
     bind(classOf[HttpEventStreamActorMetrics]).in(Scopes.SINGLETON)
   }
 
+  @Named(ModuleNames.SCHEDULER_HEARTBEAT_MONITOR)
```

Comment:
In order to align with the direction of code structures: PTAL e.g. at the `mesosphere.marathon.core.launchqueue` package:
- all the newer implementation is trying to prevent wiring in Guice: you could add a `HeartbeatModule` and use a distinct config similar to `LaunchQueueConfig` to separate wiring from config (and have a nice place to specify the default config values)
- you can/should use the `LeadershipModule.startWhenLeader` to make sure this actor is started/suspended depending on leadership status.